cli2telegram/cli.py

```diff
@@ -69,37 +76,7 @@ def cli(bot_token: str or None, chat_id: str or None, lines: Tuple[str]):
         LOGGER.warning("Message is empty, sending warning instead")
         prepared_message = f"Message is empty after processing, original message: {lines}"
 
-    _try_send_message(prepared_message)
```

Comment:
I think you deleted too much here, this line ensures the message sending when using it from the CLI

Reply:
Line 72 is changed from `_try_send_message(prepared_message)` to `_try_send_message(prepared_message, CONFIG, LOGGER, False)` because I changed the function's signature when moving it to `util`.

Reply:
Ah right, I missed that :+1: